iam-bootstrap/bootstrap-1.json

```diff
@@ -91,6 +91,19 @@
                 "*"
             ]
         },
+        {
+            "Sid": "ACMUngated",
+            "Effect": "Allow",
+            "Action": [
+                "acm:RequestCertificate",
+                "acm:DescribeCertificate",
+                "acm:GetCertificate",
+                "acm:ListTagsForCertificate"
+            ],
+            "Resource": [
+                "*"
```

Comment:
Do we need * or can we restrict to `"arn:${partition}:acm:*:${account_id}:certificate/*"`?

Reply:
It depends on how paranoid we are. (Also, I suspect that `RequestCertificate` and `ListCertificates` will need `*`).

If you think we need restrict it by resource, I can do this, but will need another round of testing.

Reply:
thats fine we can keep `*`